source/blog/2023-11-14-hanami-210.html.markdown

```diff
@@ -0,0 +1,194 @@
+---
+title: "Hanami 2.1: Views that are a sight to see"
+date: 2023-11-14 09:09 UTC
+tags: announcements
+author: Tim Riley & Luca Guidi
+image: true
+excerpt: >
+  Introducing our view layer and front end assets support.
+---
+
+After a year of work, Hanami 2.1 is here! This release introduces our view layer and front end assets support, and brings Hanami a big step closer to our full stack vision.
```

Comment:
Should we say "front-end"? I find "front end" a bit surprising.

Reply:
Yep, will do!